examples/2d-measure-accuracy.py

```diff
@@ -362,14 +410,14 @@ def modes(x):
             #    init = tf.initialize_variables(g_vars)
             #    gan.sess.run(init)
 
-            if(i > 490000):
+            if(i > 9000):
                 ax, ag, agg, dl = gan.sess.run([accuracy_x_to_g, accuracy_g_to_x, accuracy_g_to_g, gan.graph.d_log], {gan.graph.x: x_0, gan.graph.z[0]: z_0})
                 diversity += agg
                 ax_sum += ax
                 ag_sum += ag
                 dlog = dl
 
-        with open("results.csv", "a") as myfile:
```

Comment:
This example script is pretty hacky.  The tech debt impact isn't huge, but it's a really cool script and we should make it a bit easier to use.  I'll add a note to https://www.pivotaltracker.com/story/show/141733753